### PR TITLE
docs for new istio params and new auditor image params

### PR DESCRIPTION
**Pull request by scott-cotton**
## Change description

see title.

## Type of change
- [ ] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)

## Related issues

> Documentation for https://github.com/signadot/signadot/pull/3616

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Comment by daniel-de-vera** [comment]
@scott-cotton, does this still look good to you?

File: signadot/operator/README.md [hunk 1]
```diff
@@ -136,10 +136,11 @@ style resources and are not needed in an installation which uses the new
...
 ### Istio Parameters
 
-The Signadot Operator manipulates istio objects when istio VirtualServices are applied to workloads in sandboxes.  You can configure the operator to add labels and annotations to these objects when they are in use by the operator.  Note that these labels and annotations are only added if not present when the object comes into use. This can be useful for temporarily disabling CI sync, amongst other possibilities.
+When Istio is enabled (`istio.enabled: true`), the Signadot Operator manipulates Istio VirtualServices by applying new HTTPRoutes for each of the sandboxed workloads. You can configure the operator to add labels and annotations to these objects when they are in use by the operator.  Note that these labels and annotations are only added if not present when the object comes into use. This can be useful for temporarily disabling CI sync, amongst other possibilities.
```
Comment (scott-cotton) [inline signadot/operator/README.md:139]: I think this problem was there originally at my hand, but I'd suggest trying to rephrase 
`by applying new HTTPRoutes for each of the sandboxed workloads` with  something more precise/informative.  

The balance is hard to find, as being completely precise is maybe too verbose for this context.  But perhaps something along the lines of
```
by applying new HTTPRoutes where appropriate to direct traffic to sandboxed workloads
```
could work.

```

Comment (daniel-de-vera) [reply signadot/operator/README.md]: I applied the change.

**Review by daniel-de-vera — APPROVED** [review]
LGTM, but I would wait to merge until we do the operator release

File: signadot/operator/README.md [hunk 2]
```diff
@@ -132,3 +134,13 @@ style resources and are not needed in an installation which uses the new
...
+### Istio Parameters
+
+When Istio is enabled (`istio.enabled: true`), the Signadot Operator manipulates Istio VirtualServices by applying new HTTPRoutes where appropriate to direct traffic to sandboxed workloads. You can configure the operator to add labels and annotations to these objects when they are in use by the operator.  Note that these labels and annotations are only added if not present when the object comes into use. This can be useful for temporarily disabling CI sync, amongst other possibilities.
+
+
+| Name                          | Description                                        | Default |
+| ----------------------------- | -------------------------------------------------- | ------- |
+| `istio.enabled`               | Enable Istio integration                           | `false` |
+| `istio.additionalAnnotations` | Annotations to add to istio objects if not present | `{}`    |
```
Comment (foxish) [inline signadot/operator/README.md:145]: I think we can be more specific that this is `VirtualService` objects that are being modified by the signadot controllers that will get these labels and annotations.

Comment (foxish) [reply signadot/operator/README.md:145]: Ah, I see that there's the explanation below in the helm values - we can copy the same here.

Comment (scott-cotton) [reply signadot/operator/README.md]: updated to mention VirtualServices explicitly.

**Pull request merged**